pkg/dbmate/db.go

```diff
@@ -453,6 +463,32 @@ func (db *DB) Rollback() error {
 	return nil
 }
 
+func waitForOthersChangeLockToReleaseAndAcquireChangeLock(drv Driver, sqlDB *sql.DB) error {
+	counter := 0
+	for { // FIXME should we have a timeout for change locks? Or no?
+		acquiredChangeLock, err := drv.AcquireChangeLock(sqlDB)
+		if err != nil {
+			return err
+		}
+
+		if acquiredChangeLock {
+			break
+		}
+
+		if counter == 0 {
+			fmt.Print("Waiting other for database change lock to release")
+		} else {
+			fmt.Print(".")
+		}
+		time.Sleep(time.Second)
```

Comment:
if we do keep this loop here you need to make sure that ctrl+c is handled correctly

Reply:
Not sure i understand, a `time.Sleep` would terminate itself on ctrl+c no?